rpc/wrtc_call_queue_mongodb.go

```diff
@@ -922,6 +952,43 @@ func (queue *mongoDBWebRTCCallQueue) checkHostQueueSize(ctx context.Context, for
 	return errTooManyConns
 }
 
+var errOffline = status.Error(codes.Unavailable, "robot is offline or you don't have permissions to access it")
```

Comment:
> or you don't have permissions to access it

[q] How is it that the aggregation pipeline you've added for `checkHostOnline` returning no documents indicates that the caller may _not have access_ to machine?